app/src/main/java/com/hedvig/app/util/extensions/ActivityExtensions.kt

```diff
@@ -156,3 +159,21 @@ fun Activity.makeACall(uri: Uri) {
     intent.data = uri
     startActivity(intent)
 }
+
+suspend fun Activity.showReviewDialog(
+    ratingsTracker: RatingsTracker,
+    delayMillis: Long = 2000
+) {
+    delay(delayMillis)
+    val manager = ReviewManagerFactory.create(this)
+    val request = manager.requestReviewFlow()
```

Comment:
Was there a coroutine-based extension for this one in Play Core-ktx?

Reply:
Not that I'm aware of